src/api/app/components/bs_request_activity_timeline_component.html.haml

```diff
@@ -0,0 +1,24 @@
+.timeline-item
+  .row
+    .col-auto.pr-0
+      = helpers.image_tag_for(creator, size: 35, custom_class: 'rounded-circle bg-light border border-gray-400 avatars-counter')
+    .col-10
+      %p
+        %i.fas.fa-wave-square.text-dark
```

Comment:
At first glance, I didn't understand the meaning of the `wave-square` icon. I had to read the text to conclude it had to do with History.

There is a [nice icon](https://fontawesome.com/icons/rectangle-vertical-history?s=solid) for that, but it's a Pro icon :disappointed: As an alternative, we have the Free icon [clock-rotate-left](https://fontawesome.com/icons/clock-rotate-left?s=solid).
 
 Do you like the result?
 
 
![Screenshot 2022-08-04 at 16-11-57 Open Build Service](https://user-images.githubusercontent.com/2581944/182870865-5672e5af-2702-4b3f-bbca-0f4a9b0d91fb.png)

Reply:
Mmm, I agree that the `wave-square` icon is not the best, but that `clock-rorate-left` looks like going back in time, or rolling back something... I don't really like it, sorry...

Reply:
I would have liked to use [wave-pulse](https://fontawesome.com/icons/wave-pulse?s=solid), but it's a pro icon.

Reply:
> Mmm, I agree that the `wave-square` icon is not the best, but that `clock-rotate-left` looks like going back in time, or rolling back something... I don't really like it, sorry...

Yes, you're right. It seems we can revert something... Let me see if I can find something else...

Reply:
Another icon related to wave-pulse is [file-waveform](https://fontawesome.com/icons/file-waveform?s=solid), but I don't know... :unamused:

Reply:
I'm not sure what you expect, there won't be an icon perfect for something like this complex. IMHO way more important is that you choose one, people get used to it and you stick to it.

Reply:
This wasn't a review, just a comment.

Finally, @danidoni and I decided to go with the [code-commit icon](https://fontawesome.com/icons/code-commit?s=solid). New commit pushed.